cdap-hbase-compat-0.96/src/test/java/co/cask/cdap/data2/util/hbase/HTable96NameConverterTest.java

```diff
@@ -28,13 +26,7 @@
   public void testGetSysConfigTablePrefix() throws Exception {
     CConfiguration cConf = CConfiguration.create();
     String tablePrefix = cConf.get(Constants.Dataset.TABLE_PREFIX);
-
-    HBaseTableUtil tableUtil = new HBaseTableUtilFactory(cConf).get();
     HTableNameConverter hBaseNameConversionUtil = new HTableNameConverterFactory().get();
-
-    HTableDescriptorBuilder htd = tableUtil.buildHTableDescriptor(TableId.from("user", "some_table"));
```

Comment:
before this PR, the system configuration table name was derived using the table name. so if the table was named in the ``default`` namespace or non-default naming schemes, we should still be able to derive the system configuration table name (in this prefix but basically getting to that). now the system configuration table prefix cannot be derived solely based on the table name. so it basically requires simply passing in the prefix. in fact these tests are now not relevant since they are not even using anything specific to each HBase version specific HTableNameConverter class.